### ISSUE TITLE
scancode installed via pip command not working on Mac M1

### ISSUE THREAD

**Issue by rohitcoder** [nexB/scancode-toolkit#3166]
### Description
Hi, I'm trying to run this on my Mac M1, i installed from this command ``pip install scancode-toolkit`` but i'm getting below error, when i run ``scancode`` command

<img width="558" alt="image" src="https://user-images.githubusercontent.com/17665703/204994256-704af2c8-5dcf-4ea7-8dc3-39a963ab39da.png">

### How To Reproduce
1. ``pip install scancode-toolkit``
2. ``scancode``
3. You'll get error now

### System configuration

> For bug reports, it really helps us to know:

* What OS are you running on? MacOS
* What installation method was used to install/run scancode? pip

**Comment by pombredanne** [comment]
We have native C code and we need rosetta on M1 for now, though I would welcome help to compile and eventually port all of native deps to ARM. 
For now, see for instance for the workarounds in https://github.com/nexB/scancode-toolkit/blob/d64acdded0b1f9e760cb9a5e47aecffab814b811/configure#L121 and https://github.com/nexB/scancode-toolkit/commit/f6064ea8532e26ac0cd236385aa61598d29b0d66

TLDR; : either start a terminal in x86 mode or use `arch -x86_64` to the same effect.
https://scancode-toolkit.readthedocs.io/en/stable/getting-started/install.html#system-requirements

> Mac: on recent x86 64-bit macOS (10.15 and up, including 11 and 12), Use the X86 emulation mode on Apple ARM M1 CPUs.

**Comment by pombredanne** [comment]
We need to add more doc for this, but we cannot solve this short of having a free hosted CI for now. See also https://github.com/nexB/scancode-toolkit/issues/3205#issuecomment-1386666928
